testkit/gradle-testkit-plugin/src/main/kotlin/com/autonomousapps/GradleTestKitPlugin.kt

```diff
@@ -55,7 +55,7 @@ public class GradleTestKitPlugin : Plugin<Project> {
           // Gradle tests generally require more metaspace
           jvmArgs("-XX:+HeapDumpOnOutOfMemoryError", "-XX:MaxMetaspaceSize=1g")
 
-          systemProperty("com.autonomousapps.plugin-under-test.repo", extension.funcTestRepo.absolutePath)
+          systemProperty("com.autonomousapps.plugin-under-test.repo", extension.funcTestRepo.invariantSeparatorsPath)
```

Comment:
I'm not sure how this resolves the Windows issue. Could you explain? Tbh, I've never seen this property before so I looked it up, and here's what I see in the source code:
```kotlin
public val File.invariantSeparatorsPath: String
    get() = if (File.separatorChar != '/') path.replace(File.separatorChar, '/') else path
```
This looks like it forces the file path to always use `/` as element separators. Am I missing something?

Reply:
yes, that's it.
Here is the generated Groovy code on a Windows host:

```diff
-maven { url = 'E:\IdeaProjects\dependency-analysis-gradle-plugin\build\functionalTestRepo' }
+maven { url = 'E:/IdeaProjects/dependency-analysis-gradle-plugin/build/functionalTestRepo' }
```

in the current state, `\` are used but not escaped, which makes it an invalid Groovy string.
With this PR, the delimiters are replaced with `/`, which are valid Groovy strings, and is properly handled by Gradle.

Reply:
Ahhhh I see! Thank you.